dev/com.ibm.ws.microprofile.config/resources/com/ibm/ws/microprofile/config/resources/Config.nlsprops

```diff
@@ -94,6 +94,28 @@ no.such.element.CWMCG0015E=CWMCG0015E: The property {0} was not found in the con
 no.such.element.CWMCG0015E.explanation=The requested property did not exist in any of the configured sources.
 no.such.element.CWMCG0015E.useraction=Add the property to one of the existing sources or add a new source that contains the property.
 
+# Unable to cancel asynchronous update thread.
+future.update.not.cancelled.CWMCG0016E=CWMCG0016E: The server is unable to cancel the asynchronous update thread for {0}.
+future.update.not.cancelled.CWMCG0016E.explanation=The attempt to cancel a future asynchronous update failed. Typically this is caused by a long running update call to a ConfigSource instance.
+future.update.not.cancelled.CWMCG0016E.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# No implicit String constructor methods found for type: {0}
+implicit.string.constructor.method.not.found.CWMCG0017E=CWMCG0017E: No implicit String constructor methods found for type: {0}.
+implicit.string.constructor.method.not.found.CWMCG0017E.explanation=No valid String constructor methods were found on the specified class.
+implicit.string.constructor.method.not.found.CWMCG0017E.useraction=Check that the specified class has a valid String constructor method. To determine which methods are valid, refer to the MicroProfile Config specification.
+
+# Generic Type Variables are not supported. The conversion type was: {0}. The Generic Type Variable was: {1}
```

Comment:
Since this message is about generic type variables that are not supported, should this sentence be:

The {0} Generic Type Variable for the {1} conversion type for the is not supported.